src/sentry/auth/helper.py

```diff
@@ -379,6 +385,28 @@ def _post_login_redirect(self) -> HttpResponseRedirect:
 
         return response
 
+    def handle_authentication_using_verification_key(self, identity, verification_key):
+        cluster = get_redis_cluster()
```

Comment:
you can move these two lines into their own function in your other file, so that this file does not need to know those implementation details.